docs/src/userguide/runningcode.md

```diff
@@ -1,6 +1,6 @@
 # Running Code
 
-The Julia extension provides a number of different ways to run your Julia code. This section describes all these options, except how to run code in the debugger, which is covered in a separate part of the documentation.
+The Julia extension provides a number of different ways to run your Julia code. This section describes all these options, except how to run code in the debugger,found here [documentation](https://www.julia-vscode.org/docs/stable/userguide/debugging/).
```

Comment:
This link is not preferred because it always points `stable` docs. This can be resolved with `@id` and `@ref`. See [Documenter.jl document](https://documenter.juliadocs.org/dev/man/syntax/#@ref-link) for more information.